relayer/workers/parachaincommitmentrelayer/beefy-listener.go

```diff
@@ -28,11 +26,11 @@ const OUR_PARACHAIN_ID = 200
 // to the relay chain light client, but will be done once that's complete.
 
 type MessagePackage struct {
-	channelID          chainTypes.ChannelID
-	commitmentHash     types.H256
-	commitmentMessages []chainTypes.CommitmentMessage
-	paraHeadProof      string
-	mmrProof           types.GenerateMMRProofResponse
+	channelID              chainTypes.ChannelID
+	commitmentHash         types.H256
+	commitmentMessagesData types.StorageDataRaw
```

Comment:
I see you've finally come over to mine and Beefy's choice of naming, in calling the hash of the commitment the `commitmentHash`. 😆 

Nit: Can you shorten `commitmentMessagesData`? Its a bit verbose.
```suggestion
	commitmentData types.StorageDataRaw
```

Reply:
will do